molecule/python2/molecule-no-systemd.yml

```diff
@@ -16,27 +16,44 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - name: amazonlinux2
-    image: amazonlinux:2
-  - name: debian9
+  - image: amazonlinux:2
+    name: amazonlinux2
+    platform: amd64
+  - dockerfile: Dockerfile_debian_9.j2
     image: debian:stretch-slim
-    dockerfile: Dockerfile_debian_9.j2
-  - name: debian10
-    image: debian:buster-slim
-  - name: debian11
-    image: debian:bullseye-slim
-  - name: debian12
-    image: debian:bookworm-slim
-  - name: kali
-    image: kalilinux/kali-rolling
-  - name: fedora34
-    image: fedora:34
-  - name: fedora35
-    image: fedora:35
-  - name: ubuntu18
-    image: ubuntu:bionic
-  - name: ubuntu20
-    image: ubuntu:focal
+    name: debian9
+    platform: amd64
+  - image: debian:buster-slim
+    name: debian10
+    platform: amd64
+  - image: debian:bullseye-slim
+    name: debian11
+    platform: amd64
+  # There is no Python 2 package for Debian 12 and later.
+  # - image: debian:bookworm-slim
+  #   name: debian12
+  #   platform: amd64
```

Comment:
As the comment states, [there is no longer a Python 2 package for Debian 12](https://packages.debian.org/search?suite=all&searchon=names&keywords=python2).  Sid still has one, but I suspect it will be pulled.

Reply:
Should we not just update the `vars/` configuration then? I am loathe to list Debian Bookworm as a supported platform when it will break when you change an argument.

Reply:
Please see commit 716daf3.